index.php

```diff
@@ -122,7 +122,7 @@
       <div class="grid">
         <div class="two-thirds">
           <h1>Get Work Done. Or Play.</h1>
-          <p>Stay productive and focused with Multitasking View, Picture-in-Picture, Do Not Disturb, and more. Or keep work out of sight when watching videos or playing games.</p>
+          <p>Stay productive and focused with Multitasking View, Picture-in-Picture, Do Not Disturb, and more. Or keep work out of mind when watching videos or playing games.</p>
```

Comment:
This is a subtle but mindful accessibility tweak.

Reply:
These multitasking features are literally visual features that keep things out of sight, though.

Reply:
@cassidyjames Right, but keeping things out of sight is easy. The whole experience of also having no notifications, getting the blue-light dimmed and so on is less stress. You can identify with getting work out of mind.
Getting work out of sight is just a cheap trick that neither grants peace nor allows rest.
If I can use my background in media in advertising for something that isn't cynical, this is it. It sells.